lightly/api/api_workflow_datasources.py

```diff
@@ -51,44 +66,65 @@ def _download_raw_files(
                 cursor=cursor,
                 use_redirected_read_url=use_redirected_read_url,
                 **relevant_filenames_kwargs,
-                **kwargs
+                **kwargs,
             )
             cursor = response.cursor
             samples.extend(response.data)
             if progress_bar is not None:
                 progress_bar.update(len(response.data))
-        samples = [(s.file_name, s.read_url) for s in samples]
+        sample_map = {}
+        for s in samples:
+            if s.file_name.startswith("/"):
+                raise ValueError(
+                        f"Absolute file paths like {s.file_name} are not supported"
+                        f" in relevant filenames file {relevant_filenames_file_name}"
+                )
+            elif s.file_name.startswith(("./", "../")):
```

Comment:
as mentioned in my other comment this does not belong here

Reply:
See https://github.com/lightly-ai/lightly/pull/911#discussion_r953535051